### DISCUSSION TITLE
Quando faço a conexão com Rabbit exibe uma mensagem

### DISCUSSION THREAD

**Discussion by teotonio12** [php-amqplib/php-amqplib#1082] [Help wanted]
Ola, 

vou tentar descrever bem, 
sempre que vou utilizar a biblioteca eu faço a conexão com Rabbit e 
quando crio esta conexão com:

`$this->connection = new AMQPStreamConnection(env('RABBIT_HOST'),env('RABBIT_PORT'),env('RABBIT_USER'),env('RABBIT_PASSWORD'),env('RABBIT_VHOST'));`

é exibido no terminal ou nos response uma mensagem gigante e isso quebra o meu response, dando 502 e quando estou testando comandos que faça o disparo ou recebimento de mensagens fica inviável ler pq realmente sempre da essa mensagem.

Alguém sabe me dizer se existe alguma flag ou versão que não gere esta mensagem?

**Comment by lukebakken** [comment]
Please be sure that you are connecting to the correct host/ip address and port (usually 5672).

**Reply by teotonio12** [reply]
I believe that the connection does not fail, because I can send and receive messages from the queues, I only have this message when I connect.

**Comment by teotonio12** [comment]
I am closing this discussion because I found the solution, the library uses the debug env variable, if it is active, the entire connection is printed and when I debug the debug the application works normally.
